crates/turborepo-lib/src/run/task_id.rs

```diff
@@ -0,0 +1,47 @@
+pub const TASK_DELIMITER: &str = "#";
+pub const ROOT_PKG_NAME: &str = "//";
+
+pub fn get_task_id(pkg_name: impl std::fmt::Display, target: &str) -> String {
```

Comment:
I'm 99% certain that task id doesn't need to be a string. Can we try a simple struct?

Reply:
What do you mean by simple struct here?

Reply:
Thought about this a bit, and I think "simple" might not be the right word. And we maybe shouldn't block on this. I'm thinking of something like:
```rust
pub enum TaskId {
  SinglePackage { task: String },
  Monorepo {
    package: Option<String>,
    task: String
  }
}
```